docs/UserGuide.md

```diff
@@ -202,537 +237,669 @@ This subsection serves to list out the parameters used in this document. To achi
+</div>
+
+Parameters:
+
+* [`PROJECT_INDEX`](#index) The index of the project.
+* [`EVENT_INDEX`](#index) The index of an event.
+* [`DESCRIPTION`](#description) The description of an event.
+* [`DATE`](#date) The date of the event, or the starting date of the event if the event repeats every week.
+* [`TIME`](#time) The time of the event.
+* [`REPEAT_WEEKLY`](#repeat_weekly) Indicates whether the event repeats every week.
 
 Examples:
-* `updateE 1 i/1 d/Project Meeting on/24-04-2021 w/Y`
-* `updateE 2 i/2 on/14-04-2021 at/1015 w/n`
 
-#### Deleting an event from a project: `deleteE`
+* `updateE 1 i/1 d/Project Meeting on/24-04-2021 w/Y` updates the first event of the first project to be weekly event with new description `Project Meting` starting on `24-04-2021`.
+* `updateE 2 i/2 on/14-04-2021 at/1015` updates the second event of the second project to a new date `14-04-2021` at `1015`.
 
-Deletes a specified event from a specified project.
+##### Deleting an event from a project: `deleteE`
+
+Deletes an event from a project in CoLAB.
 
 Format: `deleteE PROJECT_INDEX i/EVENT_INDEX`
 
 * Deletes the event at the specified `EVENT_INDEX` from the project at the specified `PROJECT_INDEX`.
-* The `PROJECT_INDEX` refers to the number shown beside the project in the side menu.
-* The `EVENT_INDEX` refers to the number shown beside the event when viewing the project.
-* Both indexes **must be positive integers** 1, 2, 3, …​
 
-Examples:
-* `deleteE 1 i/1`
-* `deleteE 2 i/2`
+Parameters:
 
-### _**4.1.4 Groupmates**_
+* [`PROJECT_INDEX`](#index) The index of the project.
+* [`EVENT_INDEX`](#index) The index of an event.
 
-`Groupmates` contain `NAME` & `ROLE` field and are useful to keep track of `Groupmates` involved in a `Project` with their `ROLE` for the `Project`.
+Example:
 
-Each `Project` may have `Groupmates`. Hence, each of the commands related to `Groupmates` are done w.r.t the `Project` identified by `PROJECT_INDEX`.
+* `deleteE 1 i/1` deletes the first event of the first project.
 
-#### Adding a groupmate to a project: `addG`
+#### **4.1.4 Groupmates**
 
-Adds a groupmate to a specified project.
+Groupmates are used in CoLAB to represent the people you are working with for a particular project.
 
-Format: `addG PROJECT_INDEX n/NAME [r/ROLE]…​`
+![Ui](images/groupmates.png)
+
+Groupmates contain a name and zero or more roles. The role serves to indicate their role in the project.
 
-* Adds a groupmate to the project at the specified `PROJECT_INDEX`.
-* The `PROJECT_INDEX` refers to the number shown beside the project in the side menu.
-* The index **must be a positive integer** 1, 2, 3, …​
+Groupmates are sorted alphabetically by name.
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A contact can have any number of tags (including 0)
+You can use the contacts feature in CoLAB to store more information about a groupmate. The groupmate can then be easily found by searching for their name using the `findC` command.
```

Comment:
Maybe this portion can be clearer on how Contacts and Groupmates are still separate entities.

Reply:
Was a bit torn on this part. I feel since we rename it's a little more  obvious. I  didn't  want to  word this is a negative way, but rather show how this is a feature rather than a limitation. I'll spend a few mins to see if i can reword this.

Reply:
If i put this in the FAQ would it be better?

Reply:
Ooh. What about putting it as a tip?

Reply:
Adding it to FAQ can be a bonus too

Reply:
it's not a tip though is a  feature  flaw  and i don't want to highlight it  in the features. I feel naming it separately and talking about how we can find contacts using group mate  name gives  a clear indication that they are separate entities. The parameters  they take in are also different and  they can  check the FAQ if got any questions